packages/core/src/enums/MetadataModules.ts

```diff
@@ -7,19 +7,27 @@
  * with existing naming conventions)
  */
 enum MetadataModules {
+  CALIBRATION = 'calibrationModule',
   CINE = 'cineModule',
-  IMAGE_URL = 'imageUrlModule',
-  SOP_COMMON = 'sopCommonModule',
-  VOI_LUT = 'voiLutModule',
-  MODALITY_LUT = 'modalityLutModule',
+  GENERAL_IMAGE = 'generalImageModule',
   GENERAL_SERIES = 'generalSeriesModule',
-  PATIENT_STUDY = 'patientStudyModule',
-  NM_MULTIFRAME_GEOMETRY = 'nmMultiframeGeometryModule',
-  IMAGE_PLANE = 'imagePlaneModule',
+  GENERAL_STUDY = 'generalStudyModule',
   IMAGE_PIXEL = 'imagePixelModule',
+  IMAGE_PLANE = 'imagePlaneModule',
+  IMAGE_URL = 'imageUrlModule',
+  INSTANCE = 'instanceModule',
```

Comment:
changes looks much better, but since there is no `instance` module in dicom standard (if i remember correctly) and it is a construct by us (OHIF), can we just put this as `INSTANCE = 'instance'` ? and add a comment on top that this is not really a module but rather an aggregation on all others?

Reply:
and so?

```javascript
// Note: this is not a DICOM module, but this is used for gathering all the metadata for export
if (type === 'instance') {
  return getInstanceModule(imageId, metaDataProvider, instanceModuleNames);
}
```